src/components/SEP/MeetingComponents/ProposalViewModal/SEPProposalViewModal.tsx

```diff
@@ -0,0 +1,112 @@
+import AppBar from '@material-ui/core/AppBar';
+import Dialog from '@material-ui/core/Dialog';
+import DialogContent from '@material-ui/core/DialogContent';
+import Grid from '@material-ui/core/Grid';
+import IconButton from '@material-ui/core/IconButton';
+import Slide from '@material-ui/core/Slide';
+import { Theme } from '@material-ui/core/styles/createMuiTheme';
+import createStyles from '@material-ui/core/styles/createStyles';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import Toolbar from '@material-ui/core/Toolbar';
+import { TransitionProps } from '@material-ui/core/transitions/transition';
+import Typography from '@material-ui/core/Typography';
+import CloseIcon from '@material-ui/icons/Close';
+import React, { Ref } from 'react';
+
+import SimpleTabs from 'components/common/TabPanel';
+import UOLoader from 'components/common/UOLoader';
+import ProposalQuestionaryReview from 'components/review/ProposalQuestionaryReview';
+import TechnicalReviewInformation from 'components/review/TechnicalReviewInformation';
+import { useProposalData } from 'hooks/proposal/useProposalData';
+import { ContentContainer } from 'styles/StyledComponents';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    appBar: {
+      position: 'relative',
+    },
+    title: {
+      marginLeft: theme.spacing(2),
+      flex: 1,
+    },
+  })
+);
+
+const SlideComponent = (props: TransitionProps, ref: Ref<unknown>) => (
+  <Slide direction="up" ref={ref} {...props} />
+);
+
+const Transition = React.forwardRef<unknown, TransitionProps>(SlideComponent);
+
+type SEPProposalViewModalProps = {
+  proposalViewModalOpen: boolean;
+  proposalId: number;
+  setProposalViewModalOpen: (isOpen: boolean) => void;
+};
+
+const SEPProposalViewModal: React.FC<SEPProposalViewModalProps> = ({
```

Comment:
I guess it is fine for now but this is almost similar to the `ProposalReviewModal` and maybe later we can refactor a bit and have one more reusable modal where we can see proposal information that we want by passing different tabs as parameters.